src/applications/disability-benefits/view-payments/components/view-payments-lists/ViewPaymentsLists.jsx

```diff
@@ -27,23 +30,103 @@ class ViewPaymentsLists extends Component {
     this.props.getAllPayments();
   }
 
+  buildReturnedPaymentListContent = returnPayments => {
+    // If there are returned payments, set paymentsReturnedTable to a Payments component
+    // If there are NO returned payments, set paymentsReturnedTable to AlertBox
+    const paymentsReturnedTable = {};
```

Comment:
Same comment here as below. It seems like just assigning either react component to `paymentsReturnedTable` would be sufficient, unless I'm not understanding something (which could be extremely possible)